### DISCUSSION TITLE
Topojson

### DISCUSSION THREAD

**Discussion by arky** [neocarto/bertin#32] [Q&A] [answered]
How do you add topojson layers to bertin plot? Bertin seems to use geojson layers by default, can we use both geojson and topojson layers in the same plot?.  

 We have a quite a bit of topojson files that are used for maps in older d3 implementations. 

https://github.com/OpenDevelopmentMekong/odmm-data-dashboard/blob/master/images/township-v2.topojson
https://opendevelopmentmekong.github.io/odmm-data-dashboard/

**Comment by neocarto** [comment, marked as answer]
Done!
https://github.com/neocarto/bertin/commit/57b059d2d8c8669da6ec223600efeef1e6a732cc

**Reply by arky** [reply]
Awesome! Thanks @neocarto
